engines/cge2/vmenu.cpp

```diff
@@ -151,6 +158,15 @@ void VMenu::touch(uint16 mask, V2D pos, Common::KeyCode keyCode) {
 		_bar->gotoxyz(V2D(_vm, _pos2D.x, _pos2D.y + kTextVMargin + n * h - kMenuBarVerticalMargin));
 		n = _items - 1 - n;
 
+		Common::TextToSpeechManager *ttsMan = g_system->getTextToSpeechManager();
+		if (lastN != n) {
+			if (n == 0)
+				ttsMan->say(_iWantToStopPlayingAlready, Common::TextToSpeechManager::INTERRUPT);
+			else
+				ttsMan->say(_thisIsAMistakeKeepPlaying, Common::TextToSpeechManager::INTERRUPT);
+			lastN = n;
+		}
+
```

Comment:
I don't understand why you need the `_iWantToStopPlayingAlready` and `_thisIsAMistakeKeepPlaying` variables.
Could you not use `_menu[n]->_text`?

Reply:
This makes sense. I tried to do it but my compiler complained. I'll try to see again if I did anything wrong. 
```
engines/cge2/vmenu.cpp:164:12: error: no matching member function for
      call to 'say'
                        ttsMan->say(_menu[0], Common::TextToSpe...
                        ~~~~~~~~^~~
```

Reply:
You need to pass the text of the menu, not the menu itself:
```c++
ttsMan->say(_menu[n]->_text, Common::TextToSpeechManager::INTERRUPT)
```

Reply:
ok